ietf-scitt-charter.md

```diff
@@ -75,4 +75,5 @@ Milestones
 * Architecture and Terminology
 * Information and Interaction Models
 * Countersigning Format for Claim Registration
+* File-based Examples of Request-Response Interactions
 * HTTP-based REST API for Request-Response Interactions
```

Comment:
Milestones will get some attention by BoF chairs an Roman. We are providing the meat and will iterate "better phrased" milestones with them. I suggesting this consolidation:

```suggestion
* File-based Examples of Request-Response Interactions
* HTTP-based REST API for Request-Response Interactions including a critical mass of examples as implementation guidance
```

I am not entirely sure what is meant with file-based here? The examples are provided as files somewhere (out-of-scope, just do it in an official WG repo at the given time) or something... with files? Actually, I seem to have no clue. Reading the comments the abstract category seems to be "implementation guidance".

Reply:
Sounds good, this wording change addresses my concerns around completeness of documentation.

File-based was meant as a way to say something that is already in a static serialized form rather than working with a dynamic abstraction layer such as HTTP.